src/renderer.rs

```diff
@@ -511,19 +511,23 @@ fn entry_row(
                         }
 
                         @if !raw {
-                            @if let Some(size) = entry.size {
-                                span.mobile-info.size {
-                                    (maud::display(size))
-                                }
+                            span.mobile-info.size {
+                                (maud::display(&entry.size))
                             }
+                            // @if let size = entry.size {
+                            //     span.mobile-info.size {
+                            //         (maud::display(size))
+                            //     }
+                            // }
                         }
                     }
                 }
             }
             td.size-cell {
-                @if let Some(size) = entry.size {
-                    (maud::display(size))
-                }
+                (maud::display(&entry.size))
+                // @if let size = entry.size {
+                //     (maud::display(size))
+                // }
```

Comment:
What's with the commented code here?

Reply:
Ah, I didn't remove that since it was the code that would need to be switched on if we were going to throw everything behind a feature flag.